### DISCUSSION TITLE
Making language pack on OUR OWN

### DISCUSSION THREAD

**Discussion by NripPrajapat** [florisboard/florisboard#2654] [Q&A] [answered]
Can anyone share the process and tools needed to make or alter a language (or layout) from already present one like 'Chinese', or how to build one from scratch?

**Comment by KyrillGobber** [comment]
I would very mich like to know this as well. I can't seem to find any guide or repo for this

**Comment by tsiflimagas** [comment, marked as answer]
If what you have in mind is a layout that's outside of source code and you can import as an addon, then the answer is that this isn't possible yet and is scheduled for v0.5.0.
